_posts/2023-01-31-SebastianCerdan_Tribute.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: "In memoriam tribute to Professor Sebastián Cerdán"
+author: joana
+# categories: [ Jekyll, tutorial ]
+image: assets/images/tribute.jpg
+featured: true
```

Comment:
We already have 2 featured posts. I'd suggest we take one of them off before we feature this one.